airbyte-config/config-models/src/main/resources/types/AttemptSyncConfig.yaml

```diff
@@ -0,0 +1,22 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/AttemptSyncConfig.yaml
+title: AttemptSyncConfig
+description: attempt sync config
+type: object
+additionalProperties: false
+required:
+  - sourceConfiguration
+  - destinationConfiguration
+properties:
+  sourceConfiguration:
+    description: Integration specific blob. Must be a valid JSON string.
+    type: object
+    existingJavaType: com.fasterxml.jackson.databind.JsonNode
+  destinationConfiguration:
+    description: Integration specific blob. Must be a valid JSON string.
+    type: object
+    existingJavaType: com.fasterxml.jackson.databind.JsonNode
+  state:
+    description: optional state of the previous run. this object is defined per integration.
+    "$ref": State.yaml
```

Comment:
This new model, AttemptSyncConfig, will store the source/destination configuration and state per attempt